assets/js/app.js

```diff
@@ -153,26 +163,46 @@ photoUpload.onchange = function () {
   }
 };
 
-const mylist = []
-// Create a new list item when clicking on the "Add" button
-function newElement() {
-  var img = document.createElement("IMG");
-  var inputValue = document.getElementById("myInput").value;
-  document.getElementById("forms").classList.add('product-form');
-  var att = document.createAttribute("src");
-  document.getElementById('myInput').value = '';
-	att.value = inputValue;
-  img.setAttributeNode(att);
-  
-  if (inputValue === '') {
-    alert("You must write something!");
-  } else {
-    document.getElementById("results").appendChild(img);
-    const mynew = document.getElementById("results").getElementsByTagName("img")
-        // console.log(mynew[0].currentSrc) 
-        mylist.push(mynew[0].currentSrc)
-
-    console.log(mylist) 
+filterSelection("all")
+function filterSelection(c) {
+  var x, i;
+  x = document.getElementsByClassName("filterDiv");
+  if (c == "all") c = "";
+  for (i = 0; i < x.length; i++) {
+    w3RemoveClass(x[i], "show");
+    if (x[i].className.indexOf(c) > -1) w3AddClass(x[i], "show");
+  }
+}
+
+function w3AddClass(element, name) {
+  var i, arr1, arr2;
+  arr1 = element.className.split(" ");
+  arr2 = name.split(" ");
+  for (i = 0; i < arr2.length; i++) {
+    if (arr1.indexOf(arr2[i]) == -1) {
+      element.className += " " + arr2[i];
+    }
   }
-  document.getElementById("results").value = "";
 }
+
+function w3RemoveClass(element, name) {
```

Comment:
Give your functions more meaningful and descriptive names, what is `w3RemoveClass`?